gitlab/v4/objects/files.py

```diff
@@ -266,7 +267,10 @@ def raw(
         """
         file_path = utils.EncodedId(file_path)
         path = f"{self.path}/{file_path}/raw"
-        query_data = {"ref": ref}
+        if ref:
```

Comment:
I doubt people are going to call their branch `0`, but just in case:

```suggestion
        if ref is not None:
```

Reply:
Checking against `None` is better, yes.

I wanted to also catch the case where an empty string is passed, since that would be / lead to an error — maybe raising an exception would be appropriate in that case?